src/app/molecules/message/Message.jsx

```diff
@@ -479,6 +479,16 @@ const MessageOptions = React.memo(({
   const myPowerlevel = room.getMember(mx.getUserId())?.powerLevel;
   const canIRedact = room.currentState.hasSufficientPowerLevelFor('redact', myPowerlevel);
   const canSendReaction = room.currentState.maySendEvent('m.reaction', mx.getUserId());
+  // check if this has an edited event in the timeline, if so, then replace
+  // View Source with editedMEvent
+  const eventId = mEvent.getId();
+  const { editedTimeline } = roomTimeline ?? {};
+  const isEdited = roomTimeline ? editedTimeline.has(eventId) : false;
```

Comment:
this line can give an error if editedTimeline is undefined. I think we can just use `if (editedTimeline?.has(eventId))`